selfdrive/car/hyundai/values.py

```diff
@@ -1637,11 +1639,16 @@ def match_fw_to_car_fuzzy(live_fw_versions) -> Set[str]:
     (Ecu.abs, 0x7d1, None): [
       b'\xf1\x00PD ESC \x0b 104\x18\t\x03 58920-G3350',
       b'\xf1\x00PD ESC \t 104\x18\t\x03 58920-G3350',
+      b'\xf1\x00AD ESC \x11 11 \x18\x05\x06 58910-F2840',
     ],
     (Ecu.fwdRadar, 0x7d0, None): [
       b'\xf1\x00PD__ SCC F-CUP      1.00 1.00 96400-G3300         ',
       b'\xf1\x00PD__ SCC FNCUP      1.01 1.00 96400-G3000         ',
+      b'\xf1\x00AD__ SCC H-CUP      1.00 1.00 99110-F2100         ',
     ],
+    (Ecu.engine, 0x7e0, None): [
+      b'\xf1\x8161698051\x00\x00\x00\x00\x00\x00\x00\x00',
+    ]
```

Comment:
You can't add a new ECU since now the older MY cars will be missing this and not fingerprint.

Reply:
The other Elantra is an Elantra GT 2019 with no engine, so it seems split not only on MY but also model. Going to remove for now.